core/domain/exp_domain_test.py

```diff
@@ -3180,6 +3668,19 @@ def test_load_from_v25(self):
             'eid', self.YAML_CONTENT_V25)
         self.assertEqual(exploration.to_yaml(), self._LATEST_YAML_CONTENT)
 
+    def test_load_from_v26(self):
+        """Test direct loading from a v25 yaml file."""
+        exploration = exp_domain.Exploration.from_yaml(
+            'eid', self.YAML_CONTENT_V26)
+        self.assertEqual(exploration.to_yaml(), self._LATEST_YAML_CONTENT)
+
+    def test_load_from_v25_textangular(self):
```

Comment:
Make a separate test class for this to separate it from the regular yaml migration checks.

Reply:
Done!